src/go/k8s/tests/e2e/update-image-tls-client-auth-external-ca/02-assert.yaml

```diff
@@ -1,18 +1,15 @@
-apiVersion: batch/v1
-kind: Job
+apiVersion: apps/v1
+kind: Deployment
 metadata:
-  name: create-topic-update-img-external-client-ca
+  name: upd-schema-registry-mtls-console
 status:
-  conditions:
-    - status: "True"
-      type: Complete
-  succeeded: 1
+  readyReplicas: 1
```

Comment:
I know 4 tests are doing the same, but maybe it would be good to verify console custom resource?

like:
```yaml
apiVersion: redpanda.vectorized.io/v1alpha1
kind: Console
metadata:
  name: console-superusers-prefix
status:
  version: v2.2.4
  replicas: 1
  updatedReplicas: 1
  readyReplicas: 1
  availableReplicas: 1
```

REF: 
https://github.com/redpanda-data/redpanda/blob/35a0949819eee99d45f672372aee35eecc50c39c/src/go/k8s/tests/e2e/superusers-prefix/01-assert.yaml#L10-L19

Reply:
I agree. If the PR did not pass all the tests, I would change it. It take long time to complete the tests.